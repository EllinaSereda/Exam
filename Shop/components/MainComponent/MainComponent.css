body { height: 100%;} 
html { position:relative; min-height:100%; }
.MainComponent {
    font-family: 'Ubuntu', sans-serif;
    background:white;
    padding-bottom:300px;
    
}
.MainComponent .Head{
    text-align: center;
    margin:0 auto;
    
}
.MainComponent input{
    outline: none;
}
/*Шапка*/

.MainComponent .Namecent{
	background:url("https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/parfL.jpg?alt=media&token=9bb6879b-98b1-4ed3-bdad-305763783618") center no-repeat;
	background-repeat:no-repeat;
	text-align:center;
}

.MainComponent .Namecent h2 span{
	font-size:85px;
	font-family:"Grand Hotel", cursive;
	letter-spacing:0.12em;
	color:#9c3c76;
	line-height:2;
}
.MainComponent .Namecent h2{
	padding-top:15px;
}

.MainComponent .Namecent .r{
	margin-left:200px;
}
.MainComponent .sideRight{
    top:65px;
    position:absolute;
    right:10px;
}
.MainComponent .Search{
    clear:both;
    width:100%;
    height:70px;
    background-color:rgba(255, 202, 220, 0.4);
}
.MainComponent .Search input{
   width:80%;
   height:20px;
   opacity: 0.5;
   background-color:white;
   border:none;
   margin:15px;
   padding:10px;
   font-size:20px;
   border-radius:15px;
   outline: none;
}
.MainComponent .Search input:hover, .MainComponent .Search input:active {
    opacity: 1;
    outline: none;
 }
.MainComponent .SearchIcon{
    width:40px;
    padding:15px;
    float:right;
    right:40px;
}

.MainComponent .Icons{
    width:60px;
    padding:10px;
}
/*Меню*/
.MainComponent .Computer nav{

    text-align:center;
	height:70px;
	background-color:#9C3C76;
	position:relative;
}
.MainComponent .Computer nav ul li a{
    color:white;
    font-weight:bold;
    text-align: center;
    display: block;
    float:left;
    width:20%;
    font-size:22px;
    box-sizing:border-box;
    padding:10px;
}


.MainComponent .roll{
    display: inline-block;
    overflow: hidden;
    line-height: 36px;
    height: 36px;
    position: relative;
}

.MainComponent .roll span{
    padding:0;
    display: block;
    position: relative;
    top: 7px;
    transition: top .5s ease;
    -webkit-transition: top .5s ease;
    -moz-transition: top .5s ease;
}

.MainComponent .Computer .hover{
    text-shadow: #631B42 0 0 2px;
    color: #D5C3DD;
    margin-top:5px;
}
.MainComponent .Computer ul.MainComponentMenu li:hover .roll span{
    top: -35px;
}


.MainComponent .Computer .logo{
    width:200px;
    
}

.MainComponent .logoFooter{
    box-sizing: border-box;
    float:left;
    padding:30px;
    width:30%;
}
.MainComponent .logoFooter img{
    float:left;
    
    width:200px;
}
.MainComponent .FooterMenu{
    float:left;
    box-sizing: border-box;
    width:40%;
    padding:30px 0;
    text-align: center;
    font-size:18px;
    font-weight:bold;
    color:#631B42;
}
.MainComponent footer .ComputerFooter a:hover{
    color: #D5C3DD;
}

.MainComponent .ComputerFooter .thirdColomn{
    float:left;
    width:30%;
    box-sizing: border-box;
    text-transform: uppercase;
    font-size: 22px;
    font-weight:bold;
    color:#631B42;
    padding:30px 0;
    
}
.MainComponent .ComputerFooter .thirdColomn img{
    width:40px;
    margin:20px 10px;
}
.MainComponent .ComputerFooter .thirdColomn:after{
    clear:both;
    content:"";
}
.MainComponent footer {
    background-color:rgba(255, 202, 220, 0.4);
    position:absolute;
    bottom:0px;
    height:200px;
    width:100%;
}
.MainComponent footer:after{
    clear:both;
    display: block;
    content:"";
}
.MainComponent div.load{
    height:500px;
}
.MainComponent footer .AdaptiveFooter{
    display:none;
    margin:auto;
}




.MainComponent .open a{
    -webkit-transition: top 1s ease-out 0.5s;
    -moz-transition: top 1s ease-out 0.5s;
    -o-transition: top 1s ease-out 0.5s;
    transition: top 1s ease-out 0.5s;
}


.MainComponent .Adaptive{
    display:none;
}
.MainComponent .Adaptive ul a{
    text-transform: uppercase;
    background-color:rgba(156, 60, 118, 0.6);
    
}
.MainComponent .Adaptive .open a{
    font-size:18px;
    letter-spacing: 0.1em;
    display:block;
    padding:20px;
    border-bottom:1px solid rgba(156, 60, 118, 0.3);
    color:white;
}
.MainComponent .Adaptive .burg{
    background-color:rgba(156, 60, 118, 1);
}
.MainComponent .Adaptive .burg img{
   padding:10px;
    width:70px;
}
.MainComponent footer .AdaptiveFooter img{
    width:40px;
    margin:10px 30px;
}
.MainComponent footer .AdaptiveFooter span{
    float:left;
    margin:20px 40px;
    margin-right:120px;
}


/*Загрузка*/
.Loading{
    margin:200px auto;
    width:65px;
}
.lds-facebook {
    display: inline-block;
    position: relative;
    width: 64px;
    height: 64px;
  }
  .lds-facebook div {
    display: inline-block;
    position: absolute;
    left: 6px;
    width: 13px;
    background: black;
    animation: lds-facebook 1.2s cubic-bezier(0, 0.5, 0.5, 1) infinite;
  }
  .lds-facebook div:nth-child(1) {
    left: 6px;
    animation-delay: -0.24s;
  }
  .lds-facebook div:nth-child(2) {
    left: 26px;
    animation-delay: -0.12s;
  }
  .lds-facebook div:nth-child(3) {
    left: 45px;
    animation-delay: 0;
  }
  @keyframes lds-facebook {
    0% {
      top: 6px;
      height: 51px;
    }
    50%, 100% {
      top: 19px;
      height: 26px;
    }
  }
  

  
  @media screen and  (max-width:800pt) {
    .MainComponent .Namecent{
        text-align:left;
        background:url("https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/parfL.jpg?alt=media&token=9bb6879b-98b1-4ed3-bdad-305763783618") left no-repeat;
    }
    
    .MainComponent .Namecent h2 span{
        font-size:60px;
        font-family:"Grand Hotel", cursive;
        letter-spacing:0.06em;
        color:#9c3c76;
        line-height:2;
    }
    .MainComponent .Namecent h2{
        padding-top:15px;
        margin-left:130px;
    }
    
    .MainComponent .Namecent .r{
        margin-left:0;
    }
    .MainComponent .sideRight{
        top:30px;
        position:absolute;
        right:10px;
    }
    .MainComponent .sideRight img{
        width:50px;
    }
  }
  @media screen  and  (max-width:736pt) { /*от IPHONE 7 plus* горизонтально*/
    
    .MainComponent .Computer{
        display:none;
    }
    @keyframes appear {
        0% {
          display: none;
        }
        100% {
          display:block;
        }
      }
    .MainComponent .Adaptive{
        display:block;
        
    }
    
    .MainComponent{
        padding-bottom: 100px;
    }
   
    .MainComponent .close{
        display:none;
    }
    .MainComponent .open{
        display:block;

    }
    
 
    .MainComponent .Adaptive .IconsBurg{
        float:right;
        padding:10px;
        width:70px;
    }
 
    
    .MainComponent .ComputerFooter{
        display:none;
    }
    .MainComponent footer .AdaptiveFooter{
        display:block;
    }
    .MainComponent footer{
        height:60px;
        
    }
    
  }


  @media screen and (orientation:portrait) and  (max-device-width:736pt) {
    .MainComponent .Adaptive .open a{
        font-size:40px;
        padding:35px;
        
    }
    .MainComponent .Search{
        height:100px;
    }
    .MainComponent .Search input{
        box-sizing:border-box;
        width:100%;
        margin:20px 0;
        padding:15px 30px;
        font-size: 30px;
        height:60px;
        background-color:transparent;
    }
    .MainComponent .SearchIcon{
        display:none;
    }
    .MainComponent .Adaptive .burg{
        padding:30px;
    }
    .MainComponent .Adaptive .IconsBurg{
        padding:40px;
    }
  }


 