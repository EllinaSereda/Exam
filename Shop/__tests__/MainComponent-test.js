"use strict";

import React from 'react';
import renderer from 'react-test-renderer';
import MainComponent from '../components/MainComponent/MainComponent';
import { BrowserRouter } from 'react-router-dom';

test('работа Main', () => {

  // создаём тестовую версию компонента
  const component = renderer.create(
    <BrowserRouter><MainComponent user={{info:{a:{buy:{},email:"a", name:"a", order:{}}}}}/></BrowserRouter>
  );

  // получаем снэпшот (HTML-снимок) компонента для сверки, что вёрстка не испортилась
  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  // найдём в вёрстке компонента саму кнопку

  // получаем уже изменённый снэпшот
  componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

  
    
});
