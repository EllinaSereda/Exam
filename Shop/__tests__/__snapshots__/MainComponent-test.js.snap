// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа Main 1`] = `
<div
  className="MainComponent"
>
  <div
    className="Computer"
  >
    <div
      className="Head"
    >
      <div
        className="sideRight"
      >
        <a
          aria-current={null}
          className="PageLink"
          href="/account"
          onClick={[Function]}
          style={undefined}
        >
          <img
            className="Icons"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fuser-shape.png?alt=media&token=8f6275a5-6fda-46bc-b881-5602ac16e9e5"
          />
        </a>
        <a
          aria-current={null}
          className="PageLink"
          href="/order"
          onClick={[Function]}
          style={undefined}
        >
          <img
            className="Icons"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fshopping-cart-black-shape%20(3).png?alt=media&token=a7812a2b-bfc0-40fd-ac35-319abbf0d3ab"
          />
          <span
            className={null}
          />
        </a>
      </div>
      <a
        aria-current={null}
        className="PageLink"
        href="/"
        onClick={[Function]}
        style={undefined}
      >
        <div
          className="Namecent"
        >
          <h2
            className="name"
          >
            <span
              className="l"
            >
              Eau de
            </span>
            <span
              className="r"
            >
              Parfume
            </span>
          </h2>
        </div>
      </a>
    </div>
    <nav>
      <ul
        className="MainMenu"
      >
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Главная
              </span>
              <span
                className="hover"
              >
                Главная
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aaa="bbb"
            aria-current={null}
            className="PageLink"
            href="/catalog/5/1"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Каталог
              </span>
              <span
                className="hover"
              >
                Каталог
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/brands"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Бренды
              </span>
              <span
                className="hover"
              >
                Бренды
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/news"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Новости
              </span>
              <span
                className="hover"
              >
                Новости
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/about"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                О нас
              </span>
              <span
                className="hover"
              >
                О нас
              </span>
            </span>
          </a>
        </li>
      </ul>
    </nav>
  </div>
  <div
    className="Adaptive"
  >
    <nav>
      <a
        aria-current={null}
        className="PageLink"
        href="/order"
        onClick={[Function]}
        style={undefined}
      >
        <img
          className="IconsBurg"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fshopping-cart-black-shape%20(4).png?alt=media&token=139ed906-d1e3-4dda-978a-a8dd41705941"
        />
        <span
          className={null}
        />
      </a>
      <div
        className="burg"
        onClick={[Function]}
      >
        <img
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fmenu.png?alt=media&token=0674a144-2ccb-4128-9b00-fcf31006657c"
        />
      </div>
      <ul
        className="AdaptMenu close"
      >
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            Главная
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/catalog/10/1"
            onClick={[Function]}
            style={undefined}
          >
            Каталог
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/brands"
            onClick={[Function]}
            style={undefined}
          >
            Бренды
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/news"
            onClick={[Function]}
            style={undefined}
          >
            Новости
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/about"
            onClick={[Function]}
            style={undefined}
          >
            О нас
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/account"
            onClick={[Function]}
            style={undefined}
          >
            Личный кабинет
          </a>
        </li>
      </ul>
    </nav>
  </div>
  <div
    className="Search"
  >
    <a
      aria-current={null}
      className="PageLink"
      href="/catalog/10/1/s"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="SearchIcon"
        onClick={[Function]}
        src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fmagnifying-glass.png?alt=media&token=a847c8db-325c-48b9-b4c2-f0b1d348ff0a"
      />
    </a>
    <input
      onChange={[Function]}
      placeholder="Поиск"
      type="text"
    />
  </div>
  <div
    className="PagesRouter"
  />
  <footer>
    <div
      className="ComputerFooter"
    >
      <div
        className="logoFooter"
      >
        <a
          aria-current={null}
          className="PageLink"
          href="/"
          onClick={[Function]}
          style={undefined}
        >
          <img
            alt="#"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/footer.png?alt=media&token=461bc5ae-c096-4ac1-83c7-7a962079c41e"
          />
        </a>
      </div>
      <ul
        className="FooterMenu"
      >
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            Главная
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/catalog/10/1"
            onClick={[Function]}
            style={undefined}
          >
            Каталог
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/brands"
            onClick={[Function]}
            style={undefined}
          >
            Бренды
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/news"
            onClick={[Function]}
            style={undefined}
          >
            Новости
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/about"
            onClick={[Function]}
            style={undefined}
          >
            О нас
          </a>
        </li>
      </ul>
      <div
        className="thirdColomn"
      >
        <p>
          СЛЕДИТЕ ЗА НАМИ
        </p>
        <a
          href="#"
        >
          <img
            alt="vk"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fvk-social-network-logo.png?alt=media&token=a689c1c1-37ee-4add-9789-597c6a6b12bb"
          />
        </a>
        <a
          href="#"
        >
          <img
            alt="inst"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Finstagram-social-network-logo-of-photo-camera.png?alt=media&token=920e206c-c6da-4e39-8e30-5ea4126ae0d5"
          />
        </a>
        <a
          href="#"
        >
          <img
            alt="tel"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Ftelegram.png?alt=media&token=dd5a9b05-85ba-435e-a9da-73254e551450"
          />
        </a>
      </div>
    </div>
    <div
      className="AdaptiveFooter"
    >
      <span>
        СЛЕДИТЕ ЗА НАМИ
      </span>
      <a
        href="#"
      >
        <img
          alt="vk"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fvk-social-network-logo.png?alt=media&token=a689c1c1-37ee-4add-9789-597c6a6b12bb"
        />
      </a>
      <a
        href="#"
      >
        <img
          alt="inst"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Finstagram-social-network-logo-of-photo-camera.png?alt=media&token=920e206c-c6da-4e39-8e30-5ea4126ae0d5"
        />
      </a>
      <a
        href="#"
      >
        <img
          alt="tel"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Ftelegram.png?alt=media&token=dd5a9b05-85ba-435e-a9da-73254e551450"
        />
      </a>
    </div>
  </footer>
</div>
`;

exports[`работа Main 2`] = `
<div
  className="MainComponent"
>
  <div
    className="Computer"
  >
    <div
      className="Head"
    >
      <div
        className="sideRight"
      >
        <a
          aria-current={null}
          className="PageLink"
          href="/account"
          onClick={[Function]}
          style={undefined}
        >
          <img
            className="Icons"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fuser-shape.png?alt=media&token=8f6275a5-6fda-46bc-b881-5602ac16e9e5"
          />
        </a>
        <a
          aria-current={null}
          className="PageLink"
          href="/order"
          onClick={[Function]}
          style={undefined}
        >
          <img
            className="Icons"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fshopping-cart-black-shape%20(3).png?alt=media&token=a7812a2b-bfc0-40fd-ac35-319abbf0d3ab"
          />
          <span
            className={null}
          />
        </a>
      </div>
      <a
        aria-current={null}
        className="PageLink"
        href="/"
        onClick={[Function]}
        style={undefined}
      >
        <div
          className="Namecent"
        >
          <h2
            className="name"
          >
            <span
              className="l"
            >
              Eau de
            </span>
            <span
              className="r"
            >
              Parfume
            </span>
          </h2>
        </div>
      </a>
    </div>
    <nav>
      <ul
        className="MainMenu"
      >
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Главная
              </span>
              <span
                className="hover"
              >
                Главная
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aaa="bbb"
            aria-current={null}
            className="PageLink"
            href="/catalog/5/1"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Каталог
              </span>
              <span
                className="hover"
              >
                Каталог
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/brands"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Бренды
              </span>
              <span
                className="hover"
              >
                Бренды
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/news"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                Новости
              </span>
              <span
                className="hover"
              >
                Новости
              </span>
            </span>
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/about"
            onClick={[Function]}
            style={undefined}
          >
            <span
              className="roll"
            >
              <span
                className="seen"
              >
                О нас
              </span>
              <span
                className="hover"
              >
                О нас
              </span>
            </span>
          </a>
        </li>
      </ul>
    </nav>
  </div>
  <div
    className="Adaptive"
  >
    <nav>
      <a
        aria-current={null}
        className="PageLink"
        href="/order"
        onClick={[Function]}
        style={undefined}
      >
        <img
          className="IconsBurg"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fshopping-cart-black-shape%20(4).png?alt=media&token=139ed906-d1e3-4dda-978a-a8dd41705941"
        />
        <span
          className={null}
        />
      </a>
      <div
        className="burg"
        onClick={[Function]}
      >
        <img
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fmenu.png?alt=media&token=0674a144-2ccb-4128-9b00-fcf31006657c"
        />
      </div>
      <ul
        className="AdaptMenu close"
      >
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            Главная
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/catalog/10/1"
            onClick={[Function]}
            style={undefined}
          >
            Каталог
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/brands"
            onClick={[Function]}
            style={undefined}
          >
            Бренды
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/news"
            onClick={[Function]}
            style={undefined}
          >
            Новости
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/about"
            onClick={[Function]}
            style={undefined}
          >
            О нас
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/account"
            onClick={[Function]}
            style={undefined}
          >
            Личный кабинет
          </a>
        </li>
      </ul>
    </nav>
  </div>
  <div
    className="Search"
  >
    <a
      aria-current={null}
      className="PageLink"
      href="/catalog/10/1/s"
      onClick={[Function]}
      style={undefined}
    >
      <img
        className="SearchIcon"
        onClick={[Function]}
        src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fmagnifying-glass.png?alt=media&token=a847c8db-325c-48b9-b4c2-f0b1d348ff0a"
      />
    </a>
    <input
      onChange={[Function]}
      placeholder="Поиск"
      type="text"
    />
  </div>
  <div
    className="PagesRouter"
  />
  <footer>
    <div
      className="ComputerFooter"
    >
      <div
        className="logoFooter"
      >
        <a
          aria-current={null}
          className="PageLink"
          href="/"
          onClick={[Function]}
          style={undefined}
        >
          <img
            alt="#"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/footer.png?alt=media&token=461bc5ae-c096-4ac1-83c7-7a962079c41e"
          />
        </a>
      </div>
      <ul
        className="FooterMenu"
      >
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/"
            onClick={[Function]}
            style={undefined}
          >
            Главная
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/catalog/10/1"
            onClick={[Function]}
            style={undefined}
          >
            Каталог
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/brands"
            onClick={[Function]}
            style={undefined}
          >
            Бренды
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/news"
            onClick={[Function]}
            style={undefined}
          >
            Новости
          </a>
        </li>
        <li>
          <a
            aria-current={null}
            className="PageLink"
            href="/about"
            onClick={[Function]}
            style={undefined}
          >
            О нас
          </a>
        </li>
      </ul>
      <div
        className="thirdColomn"
      >
        <p>
          СЛЕДИТЕ ЗА НАМИ
        </p>
        <a
          href="#"
        >
          <img
            alt="vk"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fvk-social-network-logo.png?alt=media&token=a689c1c1-37ee-4add-9789-597c6a6b12bb"
          />
        </a>
        <a
          href="#"
        >
          <img
            alt="inst"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Finstagram-social-network-logo-of-photo-camera.png?alt=media&token=920e206c-c6da-4e39-8e30-5ea4126ae0d5"
          />
        </a>
        <a
          href="#"
        >
          <img
            alt="tel"
            src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Ftelegram.png?alt=media&token=dd5a9b05-85ba-435e-a9da-73254e551450"
          />
        </a>
      </div>
    </div>
    <div
      className="AdaptiveFooter"
    >
      <span>
        СЛЕДИТЕ ЗА НАМИ
      </span>
      <a
        href="#"
      >
        <img
          alt="vk"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Fvk-social-network-logo.png?alt=media&token=a689c1c1-37ee-4add-9789-597c6a6b12bb"
        />
      </a>
      <a
        href="#"
      >
        <img
          alt="inst"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Finstagram-social-network-logo-of-photo-camera.png?alt=media&token=920e206c-c6da-4e39-8e30-5ea4126ae0d5"
        />
      </a>
      <a
        href="#"
      >
        <img
          alt="tel"
          src="https://firebasestorage.googleapis.com/v0/b/finalproject-d5d17.appspot.com/o/fonts%2Ftelegram.png?alt=media&token=dd5a9b05-85ba-435e-a9da-73254e551450"
        />
      </a>
    </div>
  </footer>
</div>
`;
